### PR TITLE
Adding a file for Westborough, MA

### LINKED ISSUE #3: neuwib things to do

### PR DESCRIPTION
**Pull request by neuwib**
This pull request adds a file with things to do and places to eat in Westborough MA. This PR resolves #3.

### CODE REVIEW

File: neuwib-branch-hometown.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Westborough, MA
+
+### Great Places to Eat
```
Comment (chandreshsharma) [inline neuwib-branch-hometown.md]: Hi @neuwib what's the best food in Summer in Westborough, and where can I have it

File: neuwib-branch-hometown.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Fun Things to DO
```
Comment (chandreshsharma) [inline neuwib-branch-hometown.md]: @neuwib what places can I visit to enjoy the evenings

File: neuwib-branch-hometown.md (hunk 1 above)
Comment (neuwib) [inline neuwib-branch-hometown.md]: I enjoy the Westborough House of Pizza. It also has its own bar.

File: neuwib-branch-hometown.md (hunk 2 above)
Comment (neuwib) [inline neuwib-branch-hometown.md]: Sky zone! Jump around on the trampolines.

**Pull request merged**